### DISCUSSION TITLE
Is possible to automatically edit all doc versions?

### DISCUSSION THREAD

**Discussion by htessaro** [facebook/docusaurus#9473] [Q&A] [answered]
I'm creating the documentation for software that releases updates monthly. The Docusaurus approach matches almost perfectly with the project. However, due to the frequent releases, we will end up with several documentation versions. Therefore, I'm looking for an easy way to update documents from different versions if a minor issue is found, such as a typo mistake. Is there a way to do it automatically or using a centralized approach? Or the only solution is to open the document for each software version and edit it manually?

If Docusaurus does not have a solution for such a scenario, I would like to know if it is possible to use GitHub actions/workflows to work around this problem.

I was checking this [StackOverflow question](https://stackoverflow.com/questions/53962559/how-to-support-changes-in-older-versions-of-docs-maintained-by-docusaurus). However, despite describing a similar problem, a solution wasn't established.

**Comment by Josh-Cena** [comment, marked as answer]
In VSCode and many other text editors, you can easily do a workspace search-and-replace. You can also use command line, like `sed`, etc.

However, Docusaurus's philosophy is that older versions should be basically frozen. We don't backport typo fixes ourselves.
